legend-engine-xt-nonrelationalStore-mongodb-grammar-integration/src/main/java/org/finos/legend/engine/language/pure/grammar/integration/connection/MongoDBConnectionParseTreeWalker.java

```diff
@@ -43,14 +39,15 @@ public MongoDBConnectionParseTreeWalker(ParseTreeWalkerSourceInformation walkerS
         this.extension = extension;
     }
 
-    public void visitServiceStoreConnectionValue(MongoDBConnectionParserGrammar.DefinitionContext ctx, MongoDBConnection connectionValue, boolean isEmbedded)
+    public void visitMongoDBConnectionValue(MongoDBConnectionParserGrammar.DefinitionContext ctx, MongoDBConnection connectionValue, boolean isEmbedded)
     {
         // store (optional if the store is provided by embedding context, if not provided, it is required)
         MongoDBConnectionParserGrammar.ConnectionStoreContext connectionStoreContext = PureGrammarParserUtility.validateAndExtractRequiredField(ctx.connectionStore(), "store", connectionValue.sourceInformation);
         if (connectionStoreContext != null)
         {
             connectionValue.element = PureGrammarParserUtility.fromQualifiedName(connectionStoreContext.qualifiedName().packagePath() == null ? Collections.emptyList() : connectionStoreContext.qualifiedName().packagePath().identifier(), connectionStoreContext.qualifiedName().identifier());
             connectionValue.elementSourceInformation = this.walkerSourceInformation.getSourceInformation(connectionStoreContext.qualifiedName());
+            connectionValue.type = DatabaseType.MongoDb;
```

Comment:
we should move this to MongoDB instead of MongoDb ( as w have that format everywhere else)